akka-stream-tests/src/test/scala/akka/stream/scaladsl/StreamRefsSpec.scala

```diff
@@ -164,11 +164,22 @@ object StreamRefsSpec {
       actor {
         provider = remote
         serialize-messages = off
+
+        default-mailbox.mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
-      remote.netty.tcp {
-        port = ${address.getPort}
-        hostname = "${address.getHostName}"
+      remote {
+         artery {
+           enabled = on
+           transport = aeron-udp
+           canonical.hostname = "${address.getHostName}"
+           canonical.port =  ${address.getPort}
+           log-received-messages = on
+           log-sent-messages = on
+         }
+       }
+      stream.materializer.stream-ref {
+        subscription-timeout = 3 seconds
```

Comment:
Can it be covered by a test case by manually sending it a Terminated?

Reply:
Getting the right actor ref is a bit tricky... will try tho